### ISSUE TITLE
calCourseSchedule = ics() should use ics(uidDomain, prodId) to pass validation

### ISSUE THREAD

**Issue by thomashillyer** [demetrios-koziris/McGillEnhanced#30]
.ics files don't successfully import into [Google Calendar without PRODID](https://support.google.com/calendar/answer/37118#format_ical)

Using this [validator](https://icalendar.org/validator.html) shows that every event should also have a UID (although google calendar doesn't seem to actually check for this and works without it)

**Comment by thomashillyer** [comment]
From [ics.js](https://github.com/nwcell/ics.js/blob/master/ics.js) 
```
if (typeof uidDomain === 'undefined') { uidDomain = 'default'; }
  if (typeof prodId === 'undefined') { prodId = 'Calendar'; }
```
so it seems like it should work anyway but the calendar I just downloaded does not have either field and wouldn't import into GCal

**Comment by demetrios-koziris** [comment]
Thanks for opening this issue!

Looks like I'm still using an old version of ics.js so I'll update it to the current version to be able to set a PRODID and generate event UIDs.

As far as I can tell though, google calendar should still be able to import a .ics. despite no PRODID or event UIDs so I'm wondering if there's another unrelated problem. I'm assuming it's the other issue you opened (#31) but let me know if you find something else that's breaking the import.

Just to double check on that point, I created some events in google calendar, exported the calendar and removed the PRODID and UID rows from the .ics and google calendar is still able to successfully import the calendar back in. Here's a zip containing that .ics file:
[test_no_prodid_or_uid.zip](https://github.com/demetrios-koziris/McGillEnhanced/files/2761892/test_no_prodid_or_uid.zip)

**Comment by demetrios-koziris** [comment]
Should be fixed in [v3.3.72](https://github.com/demetrios-koziris/McGillEnhanced/releases/tag/v3.3.72)
